astropy/units/core.py

```diff
@@ -2164,12 +2169,15 @@ class Unit(NamedUnit, metaclass=_UnitMetaClass):
       The optional ``parse_strict`` keyword controls what happens when an
       unrecognized unit string is passed in.  It may be one of the following:
```

Comment:
We need to change this bit too: "... what happens when a unit string that does not conform to the standard or is unrecognized passed in"